app/src/main/java/com/asfoundation/wallet/util/BalanceUtils.java

```diff
@@ -36,6 +36,10 @@ public static String weiToGwei(BigDecimal wei) {
         .toPlainString();
   }
 
+  public static BigDecimal weiToGweiBigDecimal(BigDecimal wei) {
+    return Convert.fromWei(wei, Convert.Unit.GWEI);
+  }
+
   public static BigInteger gweiToWei(BigDecimal gwei) {
```

Comment:
With the introduction of the new method shouldn't we make this one call the new and then call toBigInteger?